src/chainparams.cpp

```diff
@@ -198,15 +198,16 @@ class CMainParams : public CChainParams {
             ( 3250000, uint256S("0x7f3e28bf9e309c4b57a4b70aa64d3b2ea5250ae797af84976ddc420d49684034"))
             ( 3500000, uint256S("0xeaa303b93c1c64d2b3a2cdcf6ccf21b10cc36626965cc2619661e8e1879abdfb"))
             ( 3606083, uint256S("0x954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e"))
+            ( 3854173, uint256S("0xe4b4ecda4c022406c502a247c0525480268ce7abbbef632796e8ca1646425e75"))
         };
 
         chainTxData = ChainTxData{
-            // Data as of block 954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e (height 3606083).
+            // Data as of block e4b4ecda4c022406c502a247c0525480268ce7abbbef632796e8ca1646425e75 (height 3854173).
             // Tx estimate based on average of year 2021 (~40k transactions per day)
-            1613217680, // * UNIX timestamp of last checkpoint block
-            71035468,   // * total number of transactions between genesis and last checkpoint
+            1628934997, // * UNIX timestamp of last checkpoint block
+            77616340,   // * total number of transactions between genesis and last checkpoint
                         //   (the tx=... number in the SetBestChain debug.log lines)
-            0.46        // * estimated number of transactions per second after checkpoint
+            0.31        // * estimated number of transactions per second after checkpoint
```

Comment:
`(77616340 - 71035468) / (1628934997 - 1613217680) = 0.4187019960213311`, so I think this should be `0.42`?

Reply:
I've done it off a block about a week ago, tx rate has been dropping.

Reply:
which block? the checkpoint is 660 blocks old.

Reply:
Went back ~10,080 blocks, didn't think to keep the exact details sadly though.

Reply:
Trying block #3844093, 2edc6830e6e45acd45bf1d2354ead85fd22233258c19aa48d48db6f671346bd3, tx 77432592 at 1628299921

(77616340 - 77432592) / (1628934997 - 1628299921) = 183748 635076 = 0.2893

That seem a reasonable estimate?

Reply:
```suggestion
            0.29        // * estimated number of transactions per second after checkpoint
```

Reply:
Not really. Previously we have done projections based on an entire year, now you're proposing to do it based on just a week of data. I think the sample set is too small, plus we're incentivizing an increase of on-chain transactions with this release, so I'd expect an increase in tx volume.

It's not that important though, I thought you were making a typo but if you think this is it and the last week is a good sample of what's to come in the future... then who am I to argue? Time will tell.